setup.py

```diff
@@ -42,6 +42,7 @@
     'License :: OSI Approved :: BSD License',
     'Operating System :: OS Independent',
     'Programming Language :: Python',
+    'Programming Language :: Python :: 2.6',
```

Comment:
I specifically removed this because we explicitly do not support 2.6 at this time.